docs/siem/detections/api/exceptions/api-create-exception-container.asciidoc

```diff
@@ -0,0 +1,108 @@
+
+IMPORTANT: All exception items added to the same container are evaluated using
+`OR` logic. That is, if any of the items in a container evaluate to `true`, the
+exception prevents the rule from generating an alert. Likewise, `OR` logic is
+used for evaluating exceptions when more than one exception container is
+assigned to a rule. To use the `AND` operator, you can define multiple clauses
+(`entries`) in a single exception item.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`description` |String |Describes the exception container. |Yes
+|`list_id` |String |Unique identifier. |No, automatically created when it is not
+provided.
+|`meta` |Object |Placeholder for metadata about the list container. |No
+|`name` |String |The exception container's name. |Yes
+|`namespace_type` |String a|Determines whether the exception container is available in all {kib} spaces or just the space in which it is created, where:
+
+* `single`: Only available in the {kib} space in which it is created.
+* `agnostic`: Available in all {kib} spaces.
+
+|No, defaults to `single`.
+|`tags` |String[] |String array containing words and phrases to help categorize
+exception containers. |No 
+|`type` |String a|The type of exception, which must be one of these:
+
+* `detection`: Detection rule exception
+* `endpoint`: Endpoint alert exception
+
+|Yes
+
+|==============================================
+
+===== Example requests
+
+Creates an exception container for holding trusted Linux process exception
+items:
+
+[source,console]
+--------------------------------------------------
+POST api/exception_lists
+{
+  "description": "Excludes Linux trusted processes",
+  "name": "Linux process exceptions",
+  "list_id": "trusted-linux-processes",
+  "type": "detection",
+  "namespace_type": "single",
+  "tags": [
+    "linux",
+    "processes"
+  ]
+}
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+
+==== Response payload
+
+The exception container object with a unique ID.
+
+[source,json]
+--------------------------------------------------
+{
+  "_tags": [],
+  "created_at": "2020-07-13T09:33:46.187Z",
+  "created_by": "LiverpoolFC",
+  "description": "Excludes Linux trusted processes",
+  "id": "f320c070-c4eb-11ea-80bb-11861bae2798", <1>
+  "list_id": "trusted-linux-processes",
+  "name": "Linux process exceptions",
+  "namespace_type": "single",
+  "tags": [
+    "linux",
+    "processes"
+  ],
+  "tie_breaker_id": "2c08d5a5-2ecc-4d5a-acfb-0a367f25b3f3",
+  "type": "detection",
+  "updated_at": "2020-07-13T09:33:46.359Z",
+  "updated_by": "LiverpoolFC"
+}
+--------------------------------------------------
+
+<1> The returned `id` is required to associate the exception container with
```

Comment:
We updated so that what is required to associate an exception list to a rule are `id, list_id, namespace_type, type`. We have plans to make `id` optional in the future but it's required for now. Reason for planned change is that with users often importing/exporting rules - using `list_id` ill allow them to easily exercise that functionality without generating a new list every time since `id` have to be unique.